README.md

```diff
@@ -948,21 +973,24 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [ytdl-webserver](https://github.com/Algram/ytdl-webserver) - Docker-ready webserver for downloading youtube videos. `MIT` `Nodejs`
 - [Kimai](http://www.kimai.org/) - Simple time and project tracking. ([Demo](http://www.kimai.org/demo/), [Source Code](https://github.com/kimai/kimai)) `GPL-3.0` `PHP`
 - [King Phisher](https://github.com/securestate/king-phisher) - King Phisher is a tool for testing and promoting user awareness by simulating real world phishing attacks. `BSD-3-Clause` `Python`
+- [LibreNews](https://librenews.io/) - Decentralized and secure breaking news notification system. ([Source Code](https://github.com/milesmcc/LibreNews-Server/)) `GPL-3.0` `Python`
```

Comment:
Wouldn't the https://github.com/Kickball/awesome-selfhosted#custom-communication-systems section be a better fit?

Reply:
@nodiscc Okay, I moved LibreNews to the section you suggested.